js/interface.js

```diff
@@ -555,6 +571,217 @@ function uploadFiles(files) {
   });
 }
 
+// FUNCTIONS FOR MOVING FILES/FOLDERS IN MODAL
+
+// Check if folder that has selected items to move has a subfolder and filter it
+function excludeFolders(items) {
+  var parentItemIdOfSelectedItems = navStack[navStack.length-1].id;
+  var selectedIdPlaceInModal = foldersForMoving.navStack[foldersForMoving.navStack.length-1].id;
+  var searchResult = items;
+  var selectedIdForMoving = [];
+  
+  for (var i = 0; i < selectedItemsForMoving.length; i++) {
+    if ($(selectedItemsForMoving[i]).attr('data-file-type') === 'folder') {
+      selectedIdForMoving.push(Number($(selectedItemsForMoving[i]).attr('data-id')));
+    }
+  }
+  
+  if (parentItemIdOfSelectedItems === selectedIdPlaceInModal) {
+    searchResult = searchResult.filter(function (item) {
+      for (var i = 0; i < selectedIdForMoving.length; i++) {
+        return item.id !== selectedIdForMoving[i];
+      }
+    })
+  }
+  
+  return searchResult;
+  
+}
+
+// Add app select item template
+function addAppsToSelect(apps) {
+  var $appSelectList = $('#organization-optgroup-' + apps.organizationId);
+  $appSelectList.append(templates.appsSelect(apps));
+}
+
+// Create list of folders in modal
+function addFoldersToMoveModal(res) {
```

Comment:
what's "res"? can you rename it to something more meaningful?

Reply:
fixed